examples/place-option-order.ts

```diff
@@ -43,7 +43,7 @@ async function main() {
                                     .set('second', 0)
                                     .unix()
     // Convert Unix timestamp to readable date string for use with Arrow
-    const readableExpiration = getReadableTimestamp(twoWeeksFromNow * 1000) // MM is month, DD is day, and YYYY is year
+    const readableExpiration = "08192022"
```

Comment:
Just noticed this is hard coded. Updating to dynamically select the nearest Friday expiration so this test file won't fail in the future.